uniffi_bindgen/src/bindings/python/templates/ObjectTemplate.py

```diff
@@ -14,6 +14,24 @@ def __del__(self):
         )
 
     {% for meth in obj.methods() -%}
+    {%- if meth.is_static() -%}
```

Comment:
I'm not super jazzed about the way this creates four different method implementation blocks (static void, static non-void, non-static void, non-static non-void) but I also don't have a suggestion for how to do it in a neater way.

I was *hoping* that I'd be able to just do like `{% if meth.is_static() %}@staticmethod{% endif %}`, but we also need to deal with whether or not to pass the extra `handle` as first argument. Future work might be able to consolidate the macros here into something that's a bit more general.